app/services/to_collection_form/roundtrip_validator.rb

```diff
@@ -4,7 +4,7 @@ module ToCollectionForm
   # Validates that a cocina object can be converted to a collection form and then back without loss.
   class RoundtripValidator
     def self.roundtrippable?(...)
-      new(...).call
+      new(...).roundtrippable?
```

Comment:
We tend to name the class method after the instance method.

Reply:
Then I would change the static method to `call`. When our services have a single method, that method is usually (always?) `call`.

Reply:
I suspect you're right about that. I guess what feels different about this class is it doesn't feel like a service to me, but rather a validator, so I may try to recast this as a validator, which more naturally would have a single question-mark method?

Reply:
It doesn't really work as a validator. I'm inclined to agree with you here, so I'm standardizing on `.call` for these two classes. 🍻